app/src/main/kotlin/co/netguru/baby/monitor/client/feature/analytics/UserProperty.kt

```diff
@@ -10,8 +10,12 @@ sealed class UserProperty(val value: String, val key: String) {
     class AppStateProperty(appState: AppState) :
         UserProperty(appState.name.toLowerCase(), APP_STATE_KEY)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/9907f441b3f3eb382bba2baeee9926d74f7419d8/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/analytics/UserProperty.kt#L11"></span>Implicitly using the default locale is a common source of bugs: Use <code>toLowerCase(Locale)</code> instead. For strings meant to be internal use <code>Locale.ROOT</code>, otherwise <code>Locale.getDefault()</code>.</td>
    </tr>
  </tbody>
</table>